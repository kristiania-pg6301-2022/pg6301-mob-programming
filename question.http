GET {{server}}/question/random

###
POST {{server}}/question
Content-Type: application/json

{
  "id": 976,
  "answer": "answer_b"
}

###
GET {{server}}/question/score
