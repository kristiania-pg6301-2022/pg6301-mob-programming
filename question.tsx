import * as React from "react";

export function Question() {
  return <h1>Here is a question </h1>;
}
