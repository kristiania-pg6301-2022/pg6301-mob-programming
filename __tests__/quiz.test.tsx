import * as React from "react";
import * as ReactDOM from "react-dom";
import { Question } from "../question";

describe("Quiz Application", () => {
  it("shows quiz", () => { const element = document.createElement("div")
    ReactDOM.render(<Question />, element)
    expect(element.innerHTML).toMatchSnapshot()
  });
});
