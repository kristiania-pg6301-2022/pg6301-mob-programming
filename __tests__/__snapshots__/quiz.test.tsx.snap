// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quiz Application shows quiz 1`] = `"<h1>Here is a question</h1>"`;
